packages/nextjs/pages/blockexplorer/transaction/[txHash].tsx

```diff
@@ -120,6 +121,20 @@ const TransactionPage: NextPage = () => {
                   <textarea readOnly value={transaction.input} className="p-0 textarea-primary bg-inherit h-[150px]" />
                 </td>
               </tr>
+              <tr>
+                <td>
+                  <strong>Logs:</strong>
+                </td>
+                <td>
+                  <ul>
+                    {receipt?.logs?.map((log, i) => (
+                      <li key={i}>
+                        <strong>Topics {i}:</strong> {JSON.stringify(log.topics, replacer, 2)}
```

Comment:
Maybe we do "Log {i} topics: " instead "Topics {i}"  since that `i` represents the index of log right? 

But I might be missing something 😅

Reply:
Yes! You are right! Change pushed!